core/jvm/src/main/scala/scalaz/zio/RTS.scala

```diff
@@ -635,14 +636,33 @@ private object RTS {
       }
     }
 
-    final def fork[E, A](io: IO[E, A], handler: Throwable => IO[Void, Unit]): FiberContext[E, A] = {
+    final def fork[E, A](io: IO[E, A], handler: ErrorHandler): FiberContext[E, A] = {
       val context = new FiberContext[E, A](rts, handler)
 
       rts.submit(context.evaluate(io))
 
       context
     }
 
+    private final def accumFailures: List[Throwable] => IO[Void, Unit] = {
+      case Nil => IO.now[Void, Unit](())
```

Comment:
You can use `IO.unit` to avoid allocating something here.

Reply:
Thanks!